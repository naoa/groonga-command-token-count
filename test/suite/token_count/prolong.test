register commands/token_count

table_create Entries TABLE_NO_KEY
column_create Entries body COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto
column_create Terms document_index COLUMN_INDEX|WITH_POSITION Entries body

load --table Entries
[
{"body": "データベース"}
]

token_count Terms document_index --token_size 2 --ctype ja --use_ctx_output 1
