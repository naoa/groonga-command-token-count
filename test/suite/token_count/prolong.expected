register commands/token_count
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries body COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms document_index COLUMN_INDEX|WITH_POSITION Entries body
[[0,0.0,0.0],true]
load --table Entries
[
{"body": "データベース"}
]
[[0,0.0,0.0],1]
token_count Terms document_index --token_size 2 --ctype ja --use_ctx_output 1
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      5,
      5
    ],
    [
      "デー",
      1
    ],
    [
      "ース",
      1
    ],
    [
      "タベ",
      1
    ],
    [
      "ベー",
      1
    ],
    [
      "ータ",
      1
    ]
  ]
]
